packages/material-ui-lab/src/TimelineContent/TimelineContent.js

```diff
@@ -47,11 +50,11 @@ const TimelineContent = React.forwardRef(function TimelineContent(inProps, ref)
   const props = useThemeProps({ props: inProps, name: 'MuiTimelineContent' });
   const { className, ...other } = props;
 
-  const { align = 'left' } = React.useContext(TimelineContext);
+  const { position: positionContext } = React.useContext(TimelineContext);
 
   const styleProps = {
     ...props,
-    align,
+    position: positionContext || 'left',
```

Comment:
```suggestion
    position: positionContext || 'right',
```

Isn't this correct? Seems like in `Timeline` it is the default value.